app/views/histories/index.html.erb

```diff
@@ -2,125 +2,179 @@
-              locals: { big: true,
-                        heading: "History of the UK&nbsp;government".html_safe } %>
-    <p class="intro-paragraph">In this section you can read short biographies of <a href="#notable-people">notable people</a> and explore the history of <a href="#government-buildings">government buildings</a>. You can also search our <a href="#documents-records">online records</a> and read <a href="#articles-and-blog-posts">articles and blog posts</a> by historians.</p>
+<header class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <%= render "govuk_publishing_components/components/title", {
+        title: "History of the UK&nbsp;government".html_safe
+      } %>
+    </div>
+    <div class="govuk-grid-column-two-thirds">
+      <p class="govuk-body-l">In this section you can read short biographies of <a href="#notable-people" class="govuk-link">notable people</a> and explore the history of <a href="#government-buildings" class="govuk-link">government buildings</a>. You can also search our <a href="#documents-records" class="govuk-link">online records</a> and read <a href="#articles-and-blog-posts" class="govuk-link">articles and blog posts</a> by historians.</p>
+    </div>
   </div>
 </header>
 
-<div class="block introduction">
-  <div class="inner-block floated-children">
-    <hr class="keyline">
-    <div class="column copy">
-      <div class="content">
-        <h2>Why is history important?</h2>
-        <p class="intro">Governments are remembered for their leaders and the course they set for their country. The British government has a long and fascinating history, and exploring its past can help us understand how it is run today.</p>
-        <p>The information here provides a starting point for research. It includes objective factual content and research carried out by independent and civil service historians.</p>
-      </div>
+<div class="govuk-width-container">
+  <div class="govuk-grid-row introduction">
+    <div class="govuk-grid-column-full">
+      <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--l">
     </div>
-    <div class="column image">
-      <figure class="content">
-        <%= image_tag 'history/horse-guards-vintage.jpg', alt: 'Introduction to history of government' %>
-        <figcaption>Thomas Shotter Boys &ndash; A View of the Horse Guards from Whitehall. Government Art Collection.</figcaption>
-      </figure>
+    <div class="govuk-grid-column-two-thirds">
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Why is history important?",
+        id: "why-is-history-important",
+        margin_bottom: 3,
+      } %>
+      <p class="govuk-body intro">Governments are remembered for their leaders and the course they set for their country. The British government has a long and fascinating history, and exploring its past can help us understand how it is run today.</p>
```

Comment:
I wonder if we could get rid of the `intro` class and use something from govuk-frontend instead?  The [typography](https://design-system.service.gov.uk/styles/typography/) page seems to have a few options.

Reply:
Good spot - replaced with `govuk-body-l`.